jobs/bodhi-trigger.Jenkinsfile

```diff
@@ -203,17 +203,20 @@ if (test_mode) {
     return
 }
 
-def test = null
-stage("Test") {
-    test = build(job: 'test-override', propagate: false, wait: true,
-                 parameters: [
-                    string(name: 'STREAM', value: stream),
-                    string(name: 'OVERRIDES', value: msg.update.url),
-                 ])
-}
+cosaPod(cpu: "0.1", kvm: false) {
```

Comment:
0.1 CPU enough?

Reply:
Yeah, should be plenty. We're just triggering a job and doing two HTTP POSTs essentially. :)